styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -102,6 +106,8 @@
   static final String TRIGGER_ID = "STYX_TRIGGER_ID";
   static final String TRIGGER_TYPE = "STYX_TRIGGER_TYPE";
   private static final int DEFAULT_POLL_PODS_INTERVAL_SECONDS = 60;
+  private static final int DEFAULT_POD_NON_DELETE_SECONDS = 120;
```

Comment:
This kind of thing is usually called `grace period`, could we use that here as well?

Reply:
There's already a `grace period` parameter in kubectl delete that doesn't do what we need it to do. So we opted for this naming in order to avoid confusion and people thinking they're the same thing